datasets/recipes/enum.json

```diff
@@ -0,0 +1,76 @@
+{
+  "@context": {
+    "@vocab": "https://schema.org/",
+    "sc": "https://schema.org/",
+    "ml": "http://mlcommons.org/schema/",
+    "includes": "ml:includes",
+    "recordSet": "ml:RecordSet",
+    "field": "ml:Field"
+  },
+  "@type": "sc:Dataset",
+  "name": "Enum example",
+  "description": "This is a fairly minimal example, showing a way to describe enumerations.",
+  "url": "https://example.com/dataset/enum/about",
+  "citation": "@article{nash51, author = \"Nash, John\", title  = \"Non-cooperative Games\", journal = \"Annals of Mathematics\", year = 1951, volume = \"54\", number = \"2\", pages  = \"286--295\"}",
```

Comment:
Maybe remove citation and anything which is not strictly necessary and now showcased in the example itself?